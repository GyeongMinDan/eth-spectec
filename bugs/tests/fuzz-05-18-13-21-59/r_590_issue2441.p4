 struct intrinsic_metadata_t {
        }
         struct empty_t {
      }
         control nothing<H>(inout H hdr, inout empty_t meta, in intrinsic_metadata_t imeta) {
           apply {
     }
        }
