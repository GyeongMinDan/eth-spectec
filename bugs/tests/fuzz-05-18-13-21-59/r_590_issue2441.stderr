likely_hits/r_590_issue2441.p4(5): [--Wwarn=unused] warning: Control nothing is not used; removing
         control nothing<H>(inout H hdr, inout empty_t meta, in intrinsic_metadata_t imeta) {
                 ^^^^^^^
[--Wwarn=missing] warning: Program does not contain a `main' module
